main.c

```diff
@@ -1,5 +1,5 @@
 #include <xc.h>
-#include "canlib.h"
+#include "canlib/canlib.h"
```

Comment:
why

Reply:
like i think i said in the slack thread, idk my computer just has this weird issue. probably a config file thing. but this is how i've chosen to fix it and im tired of keeping 2 versions of the code with and without the canlib/canlib.h thing